app/src/main/res/layout/promoted_story_card.xml

```diff
@@ -25,17 +25,14 @@
       android:layout_width="@{viewModel.computeLayoutWidth(), default=match_parent}"
       android:layout_height="wrap_content"
       android:onClick="@{(v) -> viewModel.clickOnStoryTile()}"
-      app:layout_constraintBottom_toBottomOf="parent"
-      app:layout_constraintEnd_toEndOf="parent"
-      app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toBottomOf="@id/recently_played_stories_text_view"
       app:roundedRectDrawableWithColor="@{viewModel.promotedStory.lessonThumbnail.backgroundColorRgb}">
 
       <org.oppia.android.app.customview.LessonThumbnailImageView
         android:id="@+id/lesson_thumbnail"
         android:layout_width="match_parent"
         android:layout_height="0dp"
         android:importantForAccessibility="no"
+        android:minWidth="280sp"
```

Comment:
Given this is 280sp, should it be tied to a specific dimension for parity with the broader promoted story card size?